Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -315,7 +318,7 @@ noncomputable def liftAlgebra [IsDomain R] [Field K] [Algebra R K]
 
 -- Porting note: had to fill in the `_` by hand for this instance
 /-- Should be introduced locally after introducing `FractionRing.liftAlgebra` -/
-theorem isScalarTower_liftAlgebra [IsDomain R] [Field K] [Algebra R K] [NoZeroSMulDivisors R K] :
+instance isScalarTower_liftAlgebra [IsDomain R] [Field K] [Algebra R K] [NoZeroSMulDivisors R K] :
```

Comment:
I think we can remove the porting note. Also the comment is now useless if this is an instance, right?

Is `FractionRing.liftAlgebra` now an instance too?

Reply:
No. `FractionRing.liftAlgebra` still cannot be an instance.